components/base/wheeled/wheeled_base.go

```diff
@@ -471,11 +461,27 @@ func (wb *wheeledBase) straightDistanceToMotorInputs(distanceMm int, mmPerSec fl
 
 // Stop commands the base to stop moving.
 func (wb *wheeledBase) Stop(ctx context.Context, extra map[string]interface{}) error {
-	var err error
-	for _, m := range wb.allMotors {
-		err = multierr.Combine(err, m.Stop(ctx, extra))
+	stopFuncs := func() []rdkutils.SimpleFunc {
+		ret := []rdkutils.SimpleFunc{}
+
+		wb.mu.Lock()
+		defer wb.mu.Unlock()
+		for _, m := range wb.left {
+			motor := m
```

Comment:
is having this line necessary? I would just call m.Stop below or change m to be named motor above

Reply:
yeah it seems unnecessary but it's actually a fix to a bug I worked on previously. I don't remember the exact reasoning but it's something about a pointer to a pointer when looping through the motors, which caused `GoFor` to be called on the same motor every time rather than on each motor